tests/test-amp-style-sanitizer.php

```diff
@@ -679,6 +679,44 @@ public function test_get_validated_url_file_path( $source, $expected, $error_cod
 		}
 	}
 
+	/**
+	 * Get data URLs.
+	 *
+	 * @returns array data: URL data.
+	 */
+	public function get_data_urls() {
+		return array(
+			'url_with_spaces' => array(
+				'html { background-image:url(url with spaces.png); }',
+				'html{background-image:url("urlwithspaces.png")}',
+			),
+		);
+	}
+
+	/**
+	 * Test handling of stylesheets with spaces in the background-image URLs.
+	 *
+	 * @dataProvider get_data_urls
+	 * @covers AMP_Style_Sanitizer::normalize_urls()
```

Comment:
I think this is covering `AMP_Style_Sanitizer::remove_spaces_from_data_urls()` now, right?